File: Backend/TrackrAPI/TrackrAPI/Dtos/Chats/ChatMensajeDTO.cs
namespace TrackrAPI.Dtos.Chats;

public class ChatMensajeDTO
{
    public int IdChatMensaje {  get; set; }
    public int IdChat {  get; set; }
    public int IdPersona { get; set; }
    public string Mensaje { get; set; }
    public DateTime Fecha { get; set; }
    public string NombrePersona { get; set; }
    public int IdArchivo { get; set; }
    public string ? Nombre { get; set; }
    public DateTime ? FechaRealizacion { get; set; }
    public string ? Archivo { get; set; }
    public string ? ArchivoTipoMime { get; set; }
    public string ? ArchivoNombre { get; set; }
    public bool ? EsVideoChat { get; set; }   
}

File: Frontend/trackr-paciente/src/app/shared/Dtos/Chat/chat-mensaje-dto.ts
export interface ChatMensajeDTO{
    idChatMensaje?: number;
    idChat: number;
    idPersona: number;
    mensaje: string;
    fecha: string;
    nombrePersona?:string;
    idArchivo:number;
    nombre?:string
    fechaRealizacion ?: string;
    archivo ?: any;
    archivoTipoMime ?: string;
    archivoNombre ?: string;
    esVideoChat ?: boolean;
    fechaYaFormateada?: boolean;
}